RELEASE.md

```diff
@@ -16,21 +16,19 @@
 5. Push the changes up as a PR
 6. Verify that the builds complete
 7. Merge the PR
-8. Fetch `master`, build and check the source packages
-    - `python -m pip install --upgrade build twine`
-    - `python -m build`
-    - Check the distributions (`twine check dist/*`, version will not yet be correct)
-9. Create a git tag in the format "vX.Y.Z"
-10. Build final packages and confirm the correct version number is being used
-     - `python -m build`
-     - Check the distributions (`twine check dist/*`)
-11. Push the Git tag
-12. Create a tagged [release on GitHub](https://github.com/pymeasure/pymeasure/releases). You'll have to paste in the changelog entry and probably edit it a bit as that form expects Markdown, not ReST (probably just removing `:code:` tags will be sufficient).
-
-## PyPI release
+8. Create a new [release on GitHub](https://github.com/pymeasure/pymeasure/releases)
+    * Add a tag name in the format "vX.Y.Z" and select "create tag on publish"
+    * You'll have to paste in the changelog entry and probably edit it a bit as that form expects Markdown, not ReST (probably just removing `:code:` tags will be sufficient).
+    * Publish the release
+8. Approve the _build and upload_ run under Actions.
+   This will create the wheel and upload it to PyPI.
+
+## PyPI release - manually
 
 Official guide [here](https://packaging.python.org/en/latest/tutorials/packaging-projects/).
 
+If the upload action does not work, you can create a PyPI release manually:
+
```

Comment:
this is missing some parts of the steps 8/9/10 above that now are gone... if the upload action does not work, you don't have a built wheel locally or on the server.

Reply:
I added them.

Reply:
thanks!